restdocs-api-spec/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
     compile(kotlin("stdlib-jdk8"))
 
     compile("org.springframework.restdocs:spring-restdocs-mockmvc:$springRestDocsVersion")
+    compile("org.springframework.restdocs:spring-restdocs-restassured:$springRestDocsVersion")
```

Comment:
Okay, but since Spring REST Docs is not restricted to Spring MockMVC wouldn't it be better to have ``restdocs-api-spec-mockmvc`` as well as ``restdocs-api-spec-restassured``?

Reply:
Good point. But this would also mean that users of our library have a breaking change because they would have to change the module in their dependencies from `restdocs-api-spec` to `restdocs-api-spec-mockmvc`. This I would like to avoid.

Reply:
Since you have never released a major version (following the schema `<major>.<minor>.<patch>`) and `0.5.0` reads like a pre-release, don't you think users would accept this _breaking_ change that is quite easy to get fixed. Alternatively you could define a `1.0` milestone and move it there? Or just leave the two classes (one for `MockMVC `and the other for `REST Assured`) in `restdocs-api-spec` and require the user to explicitly add the `spring-restdocs-restassured` dependency if desired?

Reply:
OK - convinced - so if you want to do this refactoring please go ahead with restdocs-api-spec-mockmvc as well as restdocs-api-spec-restassured.